packages/lib/delivery/src/notifications/Notifications.test.ts

```diff
@@ -1,39 +1,53 @@
 import { DeliveryInformation } from '@dm3-org/dm3-lib-messaging';
-import { NotificationChannelType } from './types';
-import { _setupNotficationBroker } from './broker/NotificationBroker';
+import { NotificationChannelType, NotificationType } from './types';
+import {
+    NotificationBroker,
+    _setupNotficationBroker,
+} from './broker/NotificationBroker';
+import { getDeliveryServiceProperties } from '../../../../backend/src/config/getDeliveryServiceProperties';
+
+jest.mock('nodemailer');
+const sendMailMock = jest.fn();
+
+const nodemailer = require('nodemailer'); //doesn't work with import. idk why
+nodemailer.createTransport.mockReturnValue({
+    sendMail: sendMailMock,
+    close: () => {},
+});
 
 describe('Notifications', () => {
     it('send notifications to channel', async () => {
-        const mailMock = jest.fn();
-
         const deliveryInformation: DeliveryInformation = {
             to: 'alice.eth',
             from: 'bob.eth',
         };
 
-        const channel1 = {
-            type: NotificationChannelType.EMAIL,
-            config: {},
-        };
+        const dsNotificationChannels =
+            getDeliveryServiceProperties().notificationChannel;
 
-        const channel2 = {
+        const channel1 = {
             type: NotificationChannelType.EMAIL,
-            config: {},
+            config: {
+                recipientValue: 'bob@gmail.com',
+                isEnabled: true,
+                isVerified: false,
```

Comment:
Can you add a test that ensures that no mail gets ended when an account is not verified?